fractal.sdk/src/main/java/com/yanchware/fractal/sdk/configuration/SdkConfiguration.java

```diff
@@ -5,6 +5,9 @@
 public interface SdkConfiguration {
```

Comment:
Should We add documentation for this too, so to explain each field what it is used for?